docs/gettsim_objects/input_variables.md

```diff
@@ -247,11 +247,35 @@ Weekly working hours of individual
 
 Type: float
 
+(elterngeld_claimed)=
+
+## `elterngeld_claimed`
+
+Individual claims Elterngeld
+
+Type: bool
+
+(elterngeld_nettoeinkommen_vorjahr_m)=
+
+## `elterngeld_nettoeinkommen_vorjahr_m`
+
+Approximation of the net wage in the 12 months before birth of youngest child (according
+to simplified calculation rules). You may let GETTSIM compute this variable via the
+`elterngeld_nettolohn_approximation_m` target in a separate run.
+
+Type: float
+
+(elterngeld_zu_verst_eink_vor_geburt_sn)=
+
+## `elterngeld_zu_verst_eink_vor_geburt_sn`
```

Comment:
What is the reason not to use `elterngeld_zu_verst_eink_vorjahr_sn` here? Has exactly the same problems as `elterngeld_nettoeinkommen_vorjahr_m`.

Name is also missing a `_y`, I guess?